src/components/app-navigator/BottomTabNavigator.tsx

```diff
@@ -80,6 +94,23 @@ type BottomTabNavigatorProps = {
 const BottomTabNavigator = ({
   onBottomTabBarLayout
 }: BottomTabNavigatorProps) => {
+  const state = useNavigationState(state => state)
+
+  if (state) {
+    let currentState: any = state
+    const routePath = []
+
+    while (
+      currentState?.routeNames?.length > 0 &&
+      currentState?.routes?.length > 0
+    ) {
+      routePath.push(currentState.routeNames[currentState.index])
+      currentState = currentState.routes[currentState.index].state
+    }
+
+    console.log({ routePath })
+  }
+
```

Comment:
This was for logging out the path for deep linking research. Will remove after discussion